src/main/java/uk/gov/beis/els/api/categories/prorefrigeratedcabinets/ProRefrigeratedCabinetsApiController.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.beis.els.api.categories.prorefrigeratedcabinets;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.beis.els.categories.internetlabelling.service.InternetLabelService;
+import uk.gov.beis.els.categories.prorefrigeratedcabinets.model.ProRefrigeratedCabinetsForm;
+import uk.gov.beis.els.categories.prorefrigeratedcabinets.service.ProRefrigeratedCabinetsService;
+import uk.gov.beis.els.model.ProductMetadata;
+import uk.gov.beis.els.service.DocumentRendererService;
+
+@RestController
+@RequestMapping("${api.v1.base_path}/professional-refrigerated-storage-cabinets")
+@Tag(name = "Professional refrigerated storage cabinets")
+public class ProRefrigeratedCabinetsApiController {
+
+  private final ProRefrigeratedCabinetsService proRefrigeratedCabinetsService;
+  private final InternetLabelService internetLabelService;
+  private final DocumentRendererService documentRendererService;
+
+  @Autowired
+  public ProRefrigeratedCabinetsApiController(
+      ProRefrigeratedCabinetsService proRefrigeratedCabinetsService,
+      InternetLabelService internetLabelService, DocumentRendererService documentRendererService) {
```

Comment:
Nit: I've spotted this strange formatting in quite a few of your controllers if you can put each param on a new line please and review the other ones.

Reply:
done